config/split_mlog_adapter.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+type splitMlogAdapter struct {
+	wrappedLog *mlog.Logger
+}
+
+func (s *splitMlogAdapter) Error(msg ...interface{}) {
+	s.wrappedLog.Error(fmt.Sprint(msg...))
+}
+
+func (s *splitMlogAdapter) Warning(msg ...interface{}) {
+	s.wrappedLog.Warn(fmt.Sprint(msg...))
+}
+
+// Ignoring more verbose messages from split
+func (s *splitMlogAdapter) Info(msg ...interface{}) {
+	//s.wrappedLog.Info(fmt.Sprint(msg...))
```

Comment:
Do we want to keep these commented lines? If not, then we can just remove them.

Reply:
Right there and easy if we have problems and need to debug.

Reply:
IMO, it's really not that hard to just write the lines ourselves. Keeping commented lines is not a good practice. 0/5.